data/case-studies/case-studies.yml

```diff
@@ -0,0 +1,523 @@
+items:
```

Comment:
@berezinant, could you please explain the difference beetwen  case-studies.yml and _case-study.example.yaml? I'd expect to have one data file not two

Reply:
There is one data file, case-studies.yml is the actual data file, and _case-study.example.yaml is the example featuring and commenting all possible fields and types of content.

As for the README.md I'll sure add there some notes on how to edit case studies